kubejs/client_scripts/JEI.js

```diff
@@ -89,6 +87,9 @@ JEIEvents.hideItems(event => {
 
     // Chipped
     event.hide(['chipped:botanist_workbench', 'chipped:glassblower', 'chipped:carpenters_table', 'chipped:loom_table', 'chipped:mason_table', 'chipped:alchemy_bench', 'chipped:tinkering_table'])
+
+    //Vanilla Milk
+    event.hide('minecraft:milk')
```

Comment:
why?

Reply:
So that there only one type of milk

Reply:
comment out the gregtech one instead, prioritize vanilla items/fluids

Reply:
got it